slangpy/tests/core/test_bitmap.py

```
@@ -248,5 +248,66 @@ def test_hdr_io(tmp_path: Path, layout: Sequence[Any]):
+    ("bc4-unorm.dds", 256, 256, Bitmap.PixelFormat.r, Bitmap.ComponentType.uint8, False),
+    ("bc5-unorm.dds", 256, 256, Bitmap.PixelFormat.rg, Bitmap.ComponentType.uint8, False),
+    ("bc6h-uf16.dds", 409, 204, Bitmap.PixelFormat.rgb, Bitmap.ComponentType.float16, False),
+    ("bc7-unorm.dds", 256, 256, Bitmap.PixelFormat.rgba, Bitmap.ComponentType.uint8, False),
+    ("bc7-unorm-srgb.dds", 256, 256, Bitmap.PixelFormat.rgba, Bitmap.ComponentType.uint8, True),
+    ("bc7-unorm-odd.dds", 127, 127, Bitmap.PixelFormat.rgba, Bitmap.ComponentType.uint8, False),
+]
+
+
+@pytest.mark.parametrize("item", DDS_READ_ITEMS, ids=[x[0] for x in DDS_READ_ITEMS])
+def test_dds_read(item: tuple[str, int, int, Bitmap.PixelFormat, Bitmap.ComponentType, bool]):
+    import slangpy.platform as platform
```

Ruff (0.15.20)

[warning] Use `from slangpy import platform` in lieu of alias

Replace with `from slangpy import platform`

(PLR0402)